src/server.cpp

```diff
@@ -2527,19 +2527,38 @@ void Server::sendMediaAnnouncement(session_t peer_id, const std::string &lang_co
 	NetworkPacket pkt(TOCLIENT_ANNOUNCE_MEDIA, 0, peer_id);
 
 	u16 media_sent = 0;
-	std::string lang_suffix;
-	lang_suffix.append(".").append(lang_code).append(".tr");
+	std::string translation_formats[3] = { ".tr", ".po", ".mo" };
+	std::string lang_suffixes[3];
+	for (size_t i = 0; i < 3; i++) {
+		lang_suffixes[i].append(".").append(lang_code).append(translation_formats[i]);
+	}
 	for (const auto &i : m_media) {
-		if (str_ends_with(i.first, ".tr") && !str_ends_with(i.first, lang_suffix))
+		bool ok = true;
+		for (size_t j = 0; j < 3; j++) {
+			if (str_ends_with(i.first, translation_formats[j]) && !str_ends_with(i.first, lang_suffixes[j])) {
```

Comment:
Could use `removeStringEnd` instead of `str_ends_with(i.first, translation_formats[j])` here